src/Serialization.js

```diff
@@ -11,8 +11,19 @@ const setter = prop => obj => value => () => obj["set_" + prop](value);
 
 exports.hashTransaction = body => () => lib.hash_transaction(body);
 
+const check_limit = (num, lower_limit, upper_limit) => {
```

Comment:
This applies to several other new functions in this PR: please change the names to use `camelCase` instead of `snake_case`, since that is the idiom in Javascript. CSL exports `snake_case` identifiers because the library itself is written in Rust which uses that casing convention (the JS bindings are generated from the Rust code)